regclient/regclient
# Add version-bump utility and pin dependencies

Signed-off-by: Brandon Mitchell <git@bmitch.net>



### Fixes issue

Pinned dependencies are easily outdated without a method to maintain them.


### Describe the change

This adds a version-bump utility to check and updated pinned and versioned dependencies.



### How to verify it

```shell
$ make util-version-check 
docker run --rm -v "/home/bmitch/data/git/regclient/:/home/bmitch/data/git/regclient/" -w "/home/bmitch/data/git/regclient" -u "1000:1000" sudobmitch/version-bump:edge check
processing file: .github/workflows/docker.yml for config .github/workflows/*.yml
processing file: .github/workflows/go.yml for config .github/workflows/*.yml
processing file: .github/workflows/inactive-cleanup.yml for config .github/workflows/*.yml
processing file: build/Dockerfile.regbot for config build/Dockerfile*
processing file: build/Dockerfile.regbot.buildkit for config build/Dockerfile*
processing file: build/Dockerfile.regctl for config build/Dockerfile*
processing file: build/Dockerfile.regctl.buildkit for config build/Dockerfile*
processing file: build/Dockerfile.regsync for config build/Dockerfile*
processing file: build/Dockerfile.regsync.buildkit for config build/Dockerfile*

$ echo $?
0
```



### Changelog text



### Please verify and check that the pull request fulfills the following requirements
